deploy/manifests/crds.yaml

```diff
@@ -544,7 +536,7 @@ spec:
                 - middleManagers
                 - routers
                 - version
-                - zookeeperReference
+                - zookeeperCluster
```

Comment:
Other operators (kafka, hdfs) seem to have standardized on  `zookeeperConfigMapName` for this.

Reply:
It's not a very good name I believe but I'll change it for consistency